packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -213,9 +213,9 @@ export async function validateOpenProjects(projects: vscode.QuickPickItem[]) {
     return projectsValidToTransform
 }
 
-export function getSha256(fileName: string) {
+export function getSha256(buffer: Buffer) {
     const hasher = crypto.createHash('sha256')
-    hasher.update(fs.readFileSync(fileName))
+    hasher.update(buffer)
     return hasher.digest('base64')
```

Comment:
does it matter if it's base64 or hex? If hex is acceptable you could use the existing util function: https://github.com/aws/aws-toolkit-vscode/blob/721e951fd291aebb47c89754b7314dd36ed01b5b/packages/core/src/shared/utilities/textUtilities.ts#L104

Reply:
It has to be `base64`. Could modify this `getStringHash` and update the ~6 uses in the codebase accordingly if we really wanted to.